src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,54 @@
+# Common data types
+
+## Preprocessed or cleaned data
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        <datatype>/
+            <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
+```
+
+Preprocessing in this context means transformations of data that do not change
+the data type of the input (as expressed by its BIDS `suffix`).
+For instance, a change in the number of dimensions is likely to disrupt the propagation
+of the input's `suffix` and generally, the outcomes of such transformation
+cannot be considered _preprocessed_ or _cleaned data_.
+Examples:
+
+ -  Motion-corrected, temporally denoised, and transformed to MNI space BOLD series
+ -  Inhomogeneity corrected and skull stripped T1w files
+ -  Motion-corrected DWI files
+ -  Time-domain filtered EEG data
+ -  MaxFilter (for example, SSS) cleaned MEG data
+
+The `space` keyword is recomended to distinguish files with different underlying
+coordinate systems or registered to different reference maps.
+The `desc` (description) keyword is a general purpose field with freeform values,
+which SHOULD be used to distinguish between multiple different versions of
+processing for the same input data.
+
+Examples of preprocessed data:
+
+```Text
+pipeline1/
+    sub-001/
+        func/
+            sub-001_task-rest_run-1_space-MNI305_bold.nii.gz
+            sub-001_task-rest_run-1_space-MNI305_bold.json
+```
+
+```Text
+pipeline1/
+    sub-001/
+        func/
+            sub-001_task-rest_run-1_desc-MC_bold.nii.gz
+            sub-001_task-rest_run-1_desc-MC_bold.json
```

Comment:
Having the suggested file names based on a specific singular component of pre-processing may be straying too much into provenance. I often see data from collaborators where every single processing step is daisy-chained onto the name of the file, which is I think not something that you would want to see in BIDS Derivatives outputs. A more general suggestion of "`_desc-preproc`" would be far more widely applicable, and is already in use e.g. `fmriprep`.

```suggestion
            sub-001_task-rest_run-1_desc-preproc_bold.nii.gz
            sub-001_task-rest_run-1_desc-preproc_bold.json
```